Services/LearningHistory/classes/class.ilLearningHistoryTimelineItem.php

```diff
@@ -75,7 +75,7 @@ public function render()
 		$f = $this->ui->factory();
 		$r = $this->ui->renderer();
 
-		$ico = $f->symbol()->icon()->custom($this->lh_entry->getIconPath(), '')->withSize(\ILIAS\UI\Component\Icon\Custom::MEDIUM);
+		$ico = $f->symbol()->icon()->custom($this->lh_entry->getIconPath(), '')->withSize(\ILIAS\UI\Component\Symbol\Icon\Icon::MEDIUM);
```

Comment:
?

Reply:
This was based on an old trunk branch, before #1890 was merged.